packages/flutter/test/widgets/list_wheel_scroll_view_test.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
Add at least one matrix check for the magnificationRatio and offAxisFraction

Reply:
Can you show me how to write these kind of tests? I have written golden tests for these params, but here you mean numerical tests right?

Reply:
ya, just a way to isolate out the effects of each of the arguments. You can copy out parts of or extend 'Scrolling, diameterRatio, perspective all changes matrix'